### PR TITLE
[SRVCOM-2097] Update the name Serverless Functions to Knative Functions in S-O description

### PR DESCRIPTION
**Pull request by skonto**
## Proposed Changes

- trivial change changing the name of funcs in csv.
-  copied the suggested text from the related Jira.

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -128,7 +128,7 @@ spec:
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
     ## Serverless functions
```
Comment (pierDipi) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:130]: Should this be Knative Functions as well?

**Review by pierDipi — COMMENTED** [review]
/cc @lance

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (skonto) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:130]: It does not hurt to fix it.

**Comment by skonto** [comment]
operator tests teardown failed in this PR too. Need to investigate.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -127,10 +127,10 @@ spec:
     - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
-    ## Serverless functions
-    Serverless functions extend Knative allowing developers to write functions that let them focus on business logic.
+    ## Knative Functions
+    Knative Functions allow developers to write functions that let them focus on business logic.
```
Comment (matzew) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:131]: allows? :shrug:

Comment (skonto) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:131]: Depends if you see it as a component yes, just copied the text from docs jira.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -149,7 +149,7 @@ spec:
     - Support for managing Knative Eventing entities: Sources, Brokers, Triggers, Channels and Subscriptions
     - A kubectl-like plugin architecture to extend the built-in functionality
     - Easy integration of Knative into Tekton pipelines by using `kn` in a Tekton task
-    - Support for creating, building and deploying serverless functions for multiple runtimes (Node, Quarkus, Golang)
+    - Support for creating, building and deploying Knative Functions for multiple runtimes (Node, Quarkus, Golang)
```
Comment (lance) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:152]: Quarkus is the only "supported" runtime. How about this?
```suggestion
    - Create, build and deploy Knative Functions for multiple runtimes, including Node.js, Quarkus, and Go.
```

File: templates/csv.yaml [hunk 4]
```diff
@@ -149,7 +149,7 @@ spec:
     - Support for managing Knative Eventing entities: Sources, Brokers, Triggers, Channels and Subscriptions
     - A kubectl-like plugin architecture to extend the built-in functionality
     - Easy integration of Knative into Tekton pipelines by using `kn` in a Tekton task
-    - Support for creating, building and deploying serverless functions for multiple runtimes (Node, Quarkus, Golang)
+    - Support for creating, building and deploying Knative Functions for multiple runtimes (Node, Quarkus, Golang)
```
Comment (lance) [inline templates/csv.yaml:152]:
```suggestion
    - Create, build and deploy Knative Functions for multiple runtimes, including Node.js, Quarkus, and Go.
```
Same as above

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (lance) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:131]: Functions, plural, is "allow". Function singular is "allows". I think when we are talking of the product, we are speaking singular (there is only one Knative Functions product). So, this is a long way of saying, "yes, it should be 'allows'". :)

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -127,10 +127,10 @@ spec:
     - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
-    ## Serverless functions
-    Serverless functions extend Knative allowing developers to write functions that let them focus on business logic.
+    ## Knative Functions
+    Knative Functions allow developers to write functions that let them focus on business logic.
     These functions are deployed as Knative Services and take advantage of Knative Serving and Eventing.
-    Serverless functions bring greater efficiency, more scalability and faster development that facilitate rapid go-to-market.
+    Knative Functions bring greater efficiency, more scalability and faster development that facilitate rapid go-to-market.
```
Comment (lance) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:133]:
```suggestion
    Knative Functions bring greater efficiency, more scalability and faster development to facilitate rapid go-to-market.
```

File: templates/csv.yaml [hunk 6]
```diff
@@ -127,10 +127,10 @@ spec:
     - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
-    ## Serverless functions
-    Serverless functions extend Knative allowing developers to write functions that let them focus on business logic.
+    ## Knative Functions
+    Knative Functions allow developers to write functions that let them focus on business logic.
```
Comment (lance) [inline templates/csv.yaml:131]: *allows

File: templates/csv.yaml [hunk 7]
```diff
@@ -127,10 +127,10 @@ spec:
...
     These functions are deployed as Knative Services and take advantage of Knative Serving and Eventing.
-    Serverless functions bring greater efficiency, more scalability and faster development that facilitate rapid go-to-market.
+    Knative Functions bring greater efficiency, more scalability and faster development that facilitate rapid go-to-market.
```
Comment (lance) [inline templates/csv.yaml:133]:
```suggestion
    Knative Functions bring greater efficiency, more scalability and faster development to facilitate rapid go-to-market.
```

**Review by lance — COMMENTED** [review]
A few minor wording suggestions

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 5 above)
Comment (skonto) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:133]: > So, this is a long way of saying, "yes, it should be 'allows'". :)

:) :+1:

**Comment by skonto** [comment]
@lance ready for another round.

**Review by lance — COMMENTED** [review]
/lgtm

**Pull request merged**